cpp/src/gandiva/precompiled/extended_math_ops_test.cc

```diff
@@ -273,4 +324,137 @@ TEST(TestExtendedMathOps, TestTrigonometricFunctions) {
   VerifyFuzzyEquals(cot_float64(M_PI / 2), tan(M_PI / 2 - M_PI / 2));
 }
 
+TEST(TestExtendedMathOps, TestAbs) {
+  // Abs functions
+  VerifyFuzzyEquals(abs_int32(0), abs(0));
+  VerifyFuzzyEquals(abs_int64(0), abs(0));
+  VerifyFuzzyEquals(abs_float32(0), abs(0));
+  VerifyFuzzyEquals(abs_float64(0), abs(0));
+
+  VerifyFuzzyEquals(abs_int32(-5), abs(-5));
+  VerifyFuzzyEquals(abs_int64(-5), abs(-5));
+  VerifyFuzzyEquals(abs_float32(-5), abs(-5));
+  VerifyFuzzyEquals(abs_float64(-5), abs(-5));
+
+  VerifyFuzzyEquals(abs_int32(-2371041), abs(-2371041));
+  VerifyFuzzyEquals(abs_int64(-2371041), abs(-2371041));
```

Comment:
Can you add a test case using `abs_int64(LONG_MIN)`?

Reply:
Test added